### ISSUE TITLE
Nicehash miner unable to connect to any pools

### ISSUE THREAD

**Issue by SNiemiec** [nicehash/NiceHashMiner-Archived#1398]
Nicehash miner cannot connect to any pools

**Comment by SNiemiec** [comment]
Nicehash was working fine on 2 different rigs.

Suddenly stopped working on Nicehash site. Miners both reported "Cannot connect to..." 

Restart. Same issues. 

Claymore connects fine and runs.

**Comment by MartinFerenec** [comment]
Yes, I have the same problem. Their servers refuse the connection. I hope its nothing serious and that they fix it as soon as possible.

**Comment by UVNVlad** [comment]
Same problem here. Even after restart.

**Comment by imransheriff** [comment]
Yea something is definitely wrong, they say it’s cuz the increased difficulty I am getting 200007 almost always, it connects and stops for sometime and connects after hours of failure
